itou/templates/search/siaes_search_results.html

```diff
@@ -53,6 +62,15 @@ <h1 class="h2">
                                 <div class="row">
                                     <div class="col-12 col-md-4">{% include "search/includes/siaes_search_filters.html" with form=form %}</div>
                                     <div class="col-12 col-md-8">
+                                        <div class="mb-3 mb-md-4">
+                                            <h3 class="h4 mb-0">
+                                                {% if request.resolver_match.view_name == "search:siaes_results" %}
+                                                    {{ siaes_count }} résultats
+                                                {% else %}
+                                                    {{ job_descriptions_count }} résultats
+                                                {% endif %}
```

Comment:
Normalement on peux [accéder au nombre total d'élément dans un objet de pagination](https://docs.djangoproject.com/en/4.2/ref/paginator/#django.core.paginator.Paginator.count) donc ceci devrais fonctionner :
```suggestion
                                                {{ results_page.count }} résultats
```